examples/cinder-csi-plugin/snapshot/example.yaml

```diff
@@ -11,6 +11,8 @@ kind: VolumeSnapshotClass
 metadata:
   name: csi-cinder-snapclass
 snapshotter: cinder.csi.openstack.org
+parameters:
+  - allow_inuse_volume_snapshot: false
```

Comment:
adding documentation of this parameter would be good.

Reply:
@jichenjc - this should be camelCase with no underscores to align with Kubernetes naming conventions (see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions). Maybe `allowInUseVolumeSnapshot`

Reply:
@vkamra that is for GO field names.
@jichenjc But , ya, we should not use `_` , it should be `-` instead . For example, https://github.com/kubernetes-csi/docs/blob/master/book/src/secrets-and-credentials-volume-snapshot-class.md#L43 .
Im ok with name `snapshot-inuse-volumes` too.

Reply:
@ramineni - you're right. `-` is the way to go.

Reply:
ok, that's reasonable , will change accordingly